README.md

```diff
@@ -1,30 +1,56 @@
-# shpeucfbackend
-SHPE UCF backend for app and website
-
-Firestore Admin doc: [Admin Firestore](https://firebase.google.com/docs/reference/admin/node/admin.firestore)
-
-How to install Firebase command Line Inference 
-
-Install the CLI by running the following npm commands:
-
-npm install --save firebase
-
-npm -g install firebase-tools
-
-Next Authorize the Firebase CLI by running the following command:
-firebase login
-![loginPic](https://github.com/SHPEUCF/shpeucfbackend/blob/master/assets/signIn.png)
-
-Your command line should give you a url so you can sign in to your firebase console so just copy and paste the url into your browser and sign in with the browser.
-Once signed in your command line should give you this message
-✔  Success! Logged in as shpeucfapps@gmail.com
-![finishedLogin](https://github.com/SHPEUCF/shpeucfbackend/blob/master/assets/onlineFirebase.png)
-
-Next cd into your functions folder then run the following command to start your firebase emulator:
-
-npm run serve
-![emulator](https://github.com/SHPEUCF/shpeucfbackend/blob/master/assets/emulatorFirebase.png)
-
-And use the following command to deploy new cloud function: 
-npm run build
-
+# SHPE @ UCF Back end Development
+
+Open-source back end development for the SHPE UCF app and website.
+
+## Contents
+* [Environment setup](#setting-up-environment)
+* [Contributing](#contributing)
+* [About](#about)
+
+## Setting up environment
+1. Clone the repository and navigate to the project.
+	```
+	git clone https://github.com/SHPEUCF/shpeucfbackend.git
+	cd shpeucfbackend
+	```
+1. Install the Firebase CLI (command line interface).
+	```
+	npm -g install firebase-tools
+	```
+1. Authorize the Firebase CLI.
+	```
+	firebase login
+	```
+	Your command line should give you a url to sign in to your firebase console. Copy and paste the url into your browser and sign in with the browser. *Ask the tech directors for the login information*.
+
+	<p align="center">
+		<img height="175" src="assets/signIn.png" />
+	</p>
+
+	Once you sign in successfully, you will get to a "Firebase CLI Login Successful" page. Woot woot! :tada:
+1. Next cd into the `functions` folder and run the following command to start your firebase emulator:
+	```
+	npm run serve
```

Comment:
Maybe you should put what npm run serve does exactly, because the actual command to start the emulator is firebase emulators:start, npm run serve does build and starts the emulator, idk might be more clear that way

Reply:
* npm run serve is a shortcut that first runs npm run build, and then runs firebase emulators:start
* firebase emulators:start starts the emulator for the firebase locally, we use this firebase emulator to test the firestore, (confusing naming scheme I know)
* npm run build, compiles the functions in the typescript code you made to work with the emulator, you need to run this every time you make a change locally and want to test your functions with the emulator
* so why use npm run serve, because you don't have to type both commands in, so yes npm run serve can be used at any time, but I think it helps to know this

Reply:
is that enough detail?